src/screens/DiscoveryScreenV2/hooks/useDiscovery.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 
+import {uniqBy} from 'lodash';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `uniqBy` from lodash is a good choice for ensuring uniqueness in the topics array. However, it's important to note that lodash functions can sometimes be heavy and impact performance. If the `discovery.initialTopics` array is not expected to be large, this should be fine. But if it could potentially contain a large number of items, consider implementing a more performant way to ensure uniqueness.
```diff
- import {uniqBy} from 'lodash';
+ import uniqBy from 'lodash/uniqBy';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->